packages/web-components/src/components/va-text-input/va-text-input.tsx

```diff
@@ -140,20 +151,20 @@ export class VaTextInput {
     const describedBy =
       `${this.ariaDescribedby} ${this.error ? 'error-message' : ''}`.trim() ||
       null; // Null so we don't add the attribute if we have an empty string
-    const inputMode =
-      this.inputmode ?
-      this.inputmode :
-      null; // Null so we don't add the attribute if we have an empty string
+    const inputMode = this.inputmode ? this.inputmode : null; // Null so we don't add the attribute if we have an empty string
     const type = this.getInputType();
 
     return (
       <Host>
-        {this.label && (
-          <label htmlFor="inputField">
-            {this.label}{' '}
-            {this.required && <span class="required">(*Required)</span>}
-          </label>
-        )}
+        <label htmlFor="inputField">
+          {this.label && (
+            <Fragment>
+              {this.label}{' '}
+              {this.required && <span class="required">(*Required)</span>}
+            </Fragment>
+          )}
+          <slot></slot>
+        </label>
```

Comment:
This was done so engineers have the option to use a label and insert content between the label and input field. Engineers can also bypass the label property to have finer control over what is rendered above the input field.

Reply:
This seems fine, holding off on approval for an a11y :+1: